.github/workflows/validate-and-report.yml

```
@@ -200,7 +211,14 @@ jobs:
             rm -rf gh-duplicates/function_calls/ || true
             mv function_calls gh-duplicates/
             mv function_graphs.md gh-duplicates
+      - name: Generate function report
+        if: matrix.version == 'pspeu'
+        run: |
+            git checkout config/
+            make -j extract
+            python3 tools/function_finder/function_finder_psp.py --use-call-trees > gh-duplicates/functions.md
```

```suggestion
            python3 tools/function_finder/function_finder_psp.py --use-call-trees > gh-duplicates-psp/functions.md
```
The output directory needs to match the `with > path` from the `Clone asset repository` step